### ISSUE TITLE
Integrate support for IMUs

### ISSUE THREAD

**Issue by puzzlepaint** [ETH3D/badslam#11]
Recent depth cameras such as the Microsoft Azure Kinect, the Occipital Structure Core, and the Intel RealSense D435i all contain an IMU. It would be very helpful for robust pose tracking to integrate an IMU term into the cost function.

**Comment by bhack** [comment]
Do you have already any specific proposal on how to fuse IMU in the cost?

**Comment by puzzlepaint** [comment]
No, since I don't plan to work on this myself in the near-term.
